### ISSUE TITLE
Training code

### ISSUE THREAD

**Issue by fakerybakery** [microsoft/VibeVoice#3]
Thank you so much for open sourcing VibeVoice - have been waiting so long for a long-form + expressive TTS model :)

Is there any chance the training code or finetuning could could be released? Would love to try finetuning on some data.

**Comment by fakerybakery** [comment]
Cc @MSLDCherryPick @YaoyaoChang

**Comment by pengzhiliang** [comment]
Hi, @fakerybakery  
Thanks for attention.
Our training code is built on a 3-rd Trainer, it may take some time to adapt to Transformers Trainer for release.
But the core diffusion loss is in [here](https://github.com/microsoft/VibeVoice/blob/b2bf6de828e1e9688c510ac73613f303155bcf8f/vibevoice/modular/modeling_vibevoice.py#L408)

**Comment by fakerybakery** [comment]
Oh cool, makes sense. I'm assuming that the trainer is internal/private for now? Is Transformers trainer on the roadmap?

Also are you on Discord? Would love to discuss :)

**Comment by zzhdbw** [comment]
same question !

**Comment by bmilde** [comment]
>  it may take some time to adapt to Transformers Trainer for release.

Are you planning to release the training code? Btw, even as is would be helpful - this is a major TTS release and I'm sure the open-source community will adapt it quickly given the amount of attention it’s receiving.

**Comment by info-wordcab** [comment]
Also interested in the training code!

**Comment by pengzhiliang** [comment]
@bmilde @zzhdbw Yeah, we plan to release it as soon as possible.
In fact, it's a pre-training model without any post-training optimization.
It's could be so cool for community to ft this model on more interesting applications.

**Comment by emptystack1024** [comment]
> same question !  同一个问题！

Hope you can make it.
